galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/StatusNames.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.framework;
+
+public class StatusNames {
```

Comment:
StatusNames ? Just have a public enum.
```
public enum TestRunStatus {
  FINISHED,
  ...etc...
}
```
Looks like you also need to store the lower-case string of each enum constant... so do this:
```
public enum TestRunStatus {
  FINISHED("finished"),
   ...etc...
  
   private String value ;

   private TestRunStatus(String value) {
     this.value = value ;
   }

   public String toString() {
      return value;
   }
}
```

Reply:
Note: TestRunStatus is more descriptive than StatusName, or statuses.

Enums generally are named in the singular also... else some code using them looks weird...
```
if (a instanceof statuses)...
```